cookiecutter.json

```diff
@@ -0,0 +1,10 @@
+{
+  "project_name": "Examplebeat",
+  "github_name": "your-github-name",
+  "beat": "{{ cookiecutter.project_name|lower }}",
+  "beat_path": "github.com/{{ cookiecutter.github_name|lower }}/{{cookiecutter.beat}}",
+  "full_name": "Nicolas Ruflin",
+  "_copy_without_render": [
+    "*.j2"
+  ]
+}
```

Comment:
@cyrilleverrier Was looking into this again. On the one hand I like fexibility, on the other hand I'm not sure if setup will be more confusing with having to set vcs* information. With the `beat_path` everyone can still overwrite it to what is needed. But I would like to encourage people to share their beats on Github, so I would keep vcs part as initially suggested.